vertical-pod-autoscaler/pkg/utils/vpa/capping.go

```diff
@@ -85,14 +85,15 @@ func (c *cappingRecommendationProcessor) Apply(
 
 // getCappedRecommendationForContainer returns a recommendation for the given container, adjusted to obey policy and limits.
 func getCappedRecommendationForContainer(
-	container apiv1.Container,
+	containerName string,
```

Comment:
Why pass both the name & container if the name is trivially accessible?

Reply:
Removed this change.